accepted/PSR-17-http-factory-meta.md

```diff
@@ -290,6 +290,12 @@ responsible for the creation of the PHP resource to use as the basis for the
 stream, and therefore also assumes responsibility for the read/write pointer
 state of the resource and resulting stream.
 
+And finally, the instances created by the `RequestFactoryInterface()`,
+`ResponseFactoryInterface()` and `ServerRequestFactoryInterface` factories
+must generate instances that return an empty body-stream in read and write
```

Comment:
I think the wording could be improved here:

> must generate instances that contain an empty `StreamInterface`, in read/write mode, as the body

Reply:
The same goes for other places where this wording is used.

Reply:
Implementations of PSR-7/17 should defer the `StreamInterface` instance creation until `getBody()` is called for the first time, since you may otherwise run out of OS file handles. (Linux has a limit of 1000 by default.)

So it doesn't have to "contain" a stream instance, and probably shouldn't - it just has to "return" one.

I had initially written "read/write" as you suggest, but changed it to "read *and* write" for clarity, since the "/" could be interpreted as "or", which would be incorrect.

But I will change the wording from "body-stream" to `StreamInterface` everywhere.